packages/cli/src/cmds/preview/preview.ts

```diff
@@ -1110,7 +1110,9 @@ export default class Preview extends YargsCommand {
     let funcEnv = undefined;
     if (func.details.binFolders !== undefined) {
       funcEnv = {
-        PATH: `${process.env.PATH}${path.delimiter}${func.details.binFolders.join(path.delimiter)}`,
+        PATH: `${path.delimiter}${func.details.binFolders.join(path.delimiter)}${process.env.PATH}${
```

Comment:
Missing delimiter between func-binFolders and env-PATH.

Reply:
Here can remove the redundant heading and ending delimiters.

Reply:
Fixed.